### ISSUE TITLE
Disabling reverse DNS lookup lists wrong host when using a bnc

### ISSUE THREAD

**Issue by steakboy7000** [glftpd/glftpd#21]
Normally, lines in login.log look something like:
```
Tue Nov 26 21:15:48 2019 [13702   ] LOGIN: *@<reverse-dns> (<real-ip>) <bnc-ip> <username> <group> <tagline>
```

When disabling reverse DNS lookup (-n 0 as command line argument) it instead looks like:

```
Tue Nov 26 21:15:48 2019 [13702   ] LOGIN: *@<bnc-ip> (<real-ip>) <bnc-ip> <username> <group> <tagline>
```

I would expect the real ip to show up as host, not the bnc ip.

**Comment by Skeddie** [comment]
I can't seem to replicate this, could you inform us which bnc is being used? Tested with simpleentry and cubnc.

**Comment by steakboy7000** [comment]
I was using cbbncd: https://cbftp.eu/cbbncd-0.11.tar.gz

It turns out that this happens because cbbncd is using IDNT incorrectly, or at least differently, which triggers this behavior in gl.

other bnc's send `IDNT <ident>@<ip>:<ip>` where the latter ip is a field intended for reverse dns.
cbbncd sends `IDNT <ident>@<ip>` and as such does not provide a revdns.
Normally glftpd handles the reverse DNS lookup so the revdns field from the bnc won't matter, but when reverse DNS lookup is disabled in gl it will instead use the revdns as provided from the bnc, and since no revdns was provided it simply uses the bnc ip. I guess this can be considered not a bug then, even though it would make more sense to use the bnc-provided ip rather than the ip of the bnc in this case.

**Comment by Skeddie** [comment]
> even though it would make more sense to use the bnc-provided ip rather than the ip of the bnc in this case.

I concur. While the matter is fixed in the latest ```cbbncd v0.12```, a fix is also provided in upcoming 2.10 BETA2.
